bootstrap.js

```diff
@@ -2,11 +2,15 @@
 
 const request = require('request-promise');
 const converter = require('rel-to-abs');
+const url = require('url');
+const util = require('util');
```

Comment:
I'd suggest removing this import and using basic concatenation for the string below.